plugins/importexport/pubmed/filter/ArticlePubMedXmlFilter.php

```diff
@@ -135,6 +138,43 @@ public function &process(&$submissions): DOMDocument
                 $articleNode->appendChild($doc->createElement('Abstract'))->appendChild($doc->createTextNode($abstract));
             }
 
+            // Keywords
+            $keywords = Repo::controlledVocab()->getBySymbolic(
+                ControlledVocab::CONTROLLED_VOCAB_SUBMISSION_KEYWORD,
+                Application::ASSOC_TYPE_PUBLICATION,
+                $publication->getId(),
+                [$publicationLocale]
+            );
+
+            if (!empty($keywords[$publicationLocale])) {
+                $objectListNode = $doc->createElement('ObjectList');
+                foreach ($keywords[$publicationLocale] as $keyword) {
+                    $objectNode = $doc->createElement('Object');
+                    $objectNode->setAttribute('Type', 'keyword');
+                    $keywordNode = $doc->createElement('Param');
+                    $keywordNode->appendChild($doc->createTextNode($keyword));
+                    $keywordNode->setAttribute('Name', 'value');
+                    $objectNode->appendChild($keywordNode);
+                    $objectListNode->appendChild($objectNode);
+                }
+                $articleNode->appendChild($objectListNode);
+            }
+
+            // References
+            $citationDao = DAORegistry::getDAO('CitationDAO'); /** @var CitationDAO $citationDao */
+            $rawCitations = $citationDao->getRawCitationsByPublicationId($publication->getId());
```

Comment:
Also here, I think now, with 3.5, you can use:
$citations = $publication->getData('citations')